src/components/flow/actions/localization/KeyLocalizationForm.tsx

```diff
@@ -61,8 +62,11 @@ export default class KeyLocalizationForm extends React.Component<
 
   private getButtons(): ButtonSet {
```

Comment:
Is there a way to not duplicate this code in every component?

Reply:
Fair point. Ya, this is a product composition over inheritance. It's harder to construct it with a helper since such a helper would need reference to the event handlers. But ya, there's a super common pattern for those dialogs, so maybe a helper function that takes a yay and a nay handler wouldn't be terrible.

Reply:
For goflow docs, that is definitely trickier. Those are loaded from a file generated from goflow, so the localizations would ideally be created there. @rowanseymour, thoughts?

Reply:
@johncordeiro we plan to add that to goflow for the 5.4 release (general idea described at https://github.com/nyaruka/goflow/issues/799)